flink-connector-kafka-e2e-tests/flink-end-to-end-tests-common-kafka/pom.xml

```diff
@@ -18,230 +18,230 @@ specific language governing permissions and limitations
 under the License.
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<parent>
-		<artifactId>flink-end-to-end-tests</artifactId>
-		<groupId>org.apache.flink</groupId>
-		<version>1.16-SNAPSHOT</version>
-	</parent>
-	<modelVersion>4.0.0</modelVersion>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.apache.flink</groupId>
+        <artifactId>flink-connector-kafka-e2e-tests</artifactId>
+        <version>3.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
 
-	<artifactId>flink-end-to-end-tests-common-kafka</artifactId>
-	<name>Flink : E2E Tests : Common Kafka</name>
+    <artifactId>flink-end-to-end-tests-common-kafka</artifactId>
```

Comment:
Inconsistent indentation with remaining poms (tabs vs spaces)

Reply:
I fixed all other pots to use spaces